mos_tests/functions/common.py

```diff
@@ -301,10 +301,12 @@ def delete_instance(nova_client, uid):
             sleep(1)
 
 
-def create_instance(nova_client, inst_name, flavor_id, net_id, security_groups,
-                    image_id='', block_device_mapping=None, timeout=5):
+def create_instance(nova_client, inst_list, inst_name, flavor_id, net_id,
```

Comment:
Yes, you are right, about the moment when I mentioned that, but what about rest of tests that depends on common.py? 
Did you check all of them?

In any case, let's wait for Timur's oppinion about this parameter, and mostly about right workflow for that case.